internal/ingress/controller/template/template.go

```diff
@@ -494,15 +493,16 @@ subs_filter '%v' '$1<base href="%v://$http_host%v">' ro;
 			// ie /something to /
 			return fmt.Sprintf(`
 rewrite (?i)%s(.*) /$1 break;
-rewrite (?i)%s / break;
+rewrite (?i)%s$ / break;
 %v%v %s%s;
 %v`, path, location.Path, xForwardedPrefix, proxyPass, proto, upstreamName, abu)
 		}
 
 		return fmt.Sprintf(`
 rewrite (?i)%s(.*) %s/$1 break;
+rewrite (?i)%s$ %s/ break;
```

Comment:
what is the reason for the target to end with slash? forgive me if it's already explained elsewhere but i can't seem to find it.

related issue: https://github.com/kubernetes/ingress-nginx/issues/1399

Reply:
Also confused.
Any target path that ends without slash cannot be matched in this way.